src/main/java/br/edu/ufcg/genus/initialization/DataLoader.java

```diff
@@ -47,16 +53,22 @@
 	
 	@Autowired
 	private EvaluationService evaluationService;
+	
+	@Autowired
+	private DiscussionService discussionService;
+	
+	@Autowired
+	private ReplyService replyService;
 
 	@Override
 	public void onApplicationEvent(ContextRefreshedEvent event) {
 
-		CreateUserInput adminInput = new CreateUserInput("administrador", "admin@gmail.com", "123456");
-		CreateUserInput prof1Input = new CreateUserInput("professor1", "prof1@gmail.com", "123456");
-		CreateUserInput prof2Input = new CreateUserInput("professor2", "prof2@gmail.com", "123456");
-		CreateUserInput prof3Input = new CreateUserInput("professor3", "prof3@gmail.com", "123456");
-		CreateUserInput stud1Input = new CreateUserInput("aluno1", "alu1@gmail.com", "123456");
-		CreateUserInput stud2Input = new CreateUserInput("aluno2", "alu2@gmail.com", "123456");
+		CreateUserInput adminInput = new CreateUserInput("Julio Silva", "admin@gmail.com", "123456");
+		CreateUserInput prof1Input = new CreateUserInput("Samuel Gomes", "prof1@gmail.com", "123456");
+		CreateUserInput prof2Input = new CreateUserInput("Bianca Alves", "prof2@gmail.com", "123456");
+		CreateUserInput prof3Input = new CreateUserInput("Marina Lima", "prof3@gmail.com", "123456");
+		CreateUserInput stud1Input = new CreateUserInput("Thiago Pinto", "alu1@gmail.com", "123456");
+		CreateUserInput stud2Input = new CreateUserInput("Amanda Martins", "alu2@gmail.com", "123456");
```

Comment:
@iaronaraujo can you please change the emails as well? Maybe something like:

"Julio Silva" email: "juliosilva88@admin.com"
"Samuel Gomes" email: "gomes_samu_freitas@prof.com"
....